pubspec.yaml

```diff
@@ -9,15 +9,16 @@ authors:
   - Flutter Community <community@flutter.zone>
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ^3.2.3
```

Comment:
Why did you change this line? We need to support as many versions of the dart SDK as we can, and the current definition is good.

Reply:
Got it reverting it back